internal/dataplane/parser/atc/field.go

```diff
@@ -80,3 +80,17 @@ func (f HTTPHeaderField) FieldType() FieldType {
 func (f HTTPHeaderField) String() string {
 	return "http.headers." + strings.ToLower(strings.ReplaceAll(f.HeaderName, "-", "_"))
 }
+
+// HTTPQueryField extracts the value of an HTTP query string from the request.
+// available since Kong 3.4.1.
```

Comment:
```suggestion
```
I think this info is redundant since KIC 3.0.0 won't start with Kong Gateway < 3.4.1

Reply:
done.